back/app/models.py

```diff
@@ -19,10 +19,23 @@ def __init__(self, name, age=None, level=None, msg=None, important=False, status
             else:
                 self.age = datetime.now().date()
 
+            self.closeDate = None  
             self.level = level
             self.msg = msg
             self.important = important
             self.status = status if status is not None else "open"
+            if createDate:
+                self.createDate = datetime.fromisoformat(createDate)
```

Comment:
this might return `Wed, 12 May 2004 12:12:12 GMT` which is not an expected return!